src/Stormpath.SDK.Abstractions/Oauth/ClientCredentialsGrantRequest.cs

```diff
@@ -0,0 +1,28 @@
+namespace Stormpath.SDK.Oauth
+{
+    /// <summary>
+    /// A request to exchange a client_id and client_secret for an OAuth 2.0 access token.
+    /// </summary>
+    public sealed class ClientCredentialsGrantRequest : AbstractOauthGrantRequest
+    {
+        /// <summary>
+        /// Creates a new <see cref="ClientCredentialsGrantRequest"/> instance.
+        /// </summary>
+        public ClientCredentialsGrantRequest()
+            : base("client_credentials")
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the client id.
+        /// </summary>
+        /// <value>Client ID.</value>
+        public string ClientID { get; set; }
```

Comment:
You'll need to use `[SerializedProperty(Name="client_id")]` to get it to serialize correctly - see `RefreshTokenGrantRequest` for an example